site/docs/spark-queries.md

```diff
@@ -219,9 +221,28 @@ SELECT * FROM prod.db.table.files
 +-------------------------------------------------------------------------+-------------+--------------+--------------------+--------------------+------------------+-------------------+------------------+-----------------+-----------------+--------------+---------------+
 ```
 
+### All Data Files
+
+To show a table's valid data files and each file's metadata, run:
+> tip: a valid data file is one that is readable from any snapshot currently tracked by the table.
```

Comment:
notes in the doc are in the form of `!!! Note`, example: https://github.com/apache/iceberg/blob/master/site/docs/spark-writes.md?plain=1#L298-L301

Also we should add the warning that this table may return duplicate rows.